dataline-workers/src/main/java/io/dataline/workers/protocols/singer/DefaultSingerTap.java

```diff
@@ -110,9 +111,9 @@ public void start(StandardTapConfig input, Path jobRoot) throws Exception {
       cmd = ArrayUtils.addAll(cmd, "--state", WorkerConstants.INPUT_STATE_JSON_FILENAME);
     }
 
-    tapProcess = pbf.create(jobRoot, imageName, cmd)
-        .redirectError(jobRoot.resolve(WorkerConstants.TAP_ERR_LOG).toFile())
-        .start();
+    tapProcess = pbf.create(jobRoot, imageName, cmd).start();
+    // stdout logs are logged elsewhere since stdout also contains data
```

Comment:
are they actually logged elsewhere? or they effective just not logged? i suppose we can add to `SingerJsonStreamFactory` that it logs anything that isn't valid json. it's quite hack, but might give us better visibility. all of this can be handled separate from this PR though.

Reply:
yep, I mentioned that in the pr text. I will handle it separately